### PR TITLE
Update StartMenuL10N.txt - add Open Pinned string for translation

### PR DESCRIPTION
**Pull request by vodek3**

### CODE REVIEW

File: Src/Localization/Persian/StartMenuL10N.txt [hunk 1]
```diff
@@ -79,6 +79,7 @@ Menu.Open = با&ز کردن
 Menu.OpenAll = باز کردن تمام &کاربرها
 Menu.Explore = کاو&ش
 Menu.ExploreAll = کاوش ت&مام کاربرها
+Menu.OpenPinned = Open Pinned
```
Comment (ge0rdi) [inline Src/Localization/Persian/StartMenuL10N.txt:82]: What is the point of this change?
This file is not used for anything.

Comment (vodek3) [reply Src/Localization/Persian/StartMenuL10N.txt:82]: Предлагаю удалить его!

Comment (ge0rdi) [reply Src/Localization/Persian/StartMenuL10N.txt:82]: Please, write comments in english.

**Comment by ge0rdi** [comment]
Closing, as this PR doesn't make any sense.
There is no point to change random files in `Localization` folder as they aren't used for anything.

**Pull request closed without merging**